cli/cmd/inject.go

```diff
@@ -54,6 +55,7 @@ func newInjectOptions() *injectOptions {
 
 func newCmdInject() *cobra.Command {
 	options := newInjectOptions()
+	var advancedOption bool
```

Comment:
Right. And also it's just used in `newCmdInject` since it doesn't need to be validated or persisted elsewhere, so I figured I'd just make it a local var.

Reply:
Ahh makes sense. Thanks for the explanation.

Reply:
btw your suggestion about `rt.injectProxy` makes sense; I'll do that in the next push :+1:

Reply:
Actually while implementing that change I realized uninject.go also makes use of `inject.NewResourceConfig` and the resulting `ResourceConfig` but just for parsing the workload (it doesn't call `conf.GetPatch`). It is only when `conf.GetPatch` is called that we perform the injection, so I'd rather leave that argument there in the only function that uses it, and try keeping `ResourceConfig` as succinct as possible.

Reply:
Good point. I didn't realize that same object was used for `uninject` as well having the argument separate makes more sense.